packages/rashell/rashell.0.2.1/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/michipili/rashell/releases/download/v0.2.0/rashell-0.2.0.tar.xz"
+checksum: "7a1afd61c5124443d3c9e719db172ef1"
```

Comment:
The checksum [is wrong](https://travis-ci.org/ocaml/opam-repository/jobs/101775577):

```
[ERROR] Bad checksum for
        /Users/travis/.opam/packages.dev/rashell.0.2.1/rashell-0.2.0.tar.xz:
          - 7a1afd61c5124443d3c9e719db172ef1 [expected result]
          - 204dd28b3720d87b0dd9ddf78c888b3e [actual result]
        This may be fixed by running `opam update`.
```